.codecov.yml

```diff
@@ -6,11 +6,14 @@ coverage:
   status:
     project:
       default:
-        target: auto
+        target: null
+        threshold: null
```

Comment:
Hmm, why `null`? The documentation seems to suggest that `no` should be used? The codecov.yml validator also rejects it.